### PR TITLE
crystal: update to 1.3.2

### PR DESCRIPTION
**Pull request by mohd-akram**
###### Tested on
macOS 12.1 21C52 x86_64
Xcode 13.2.1 13C100

###### Verification
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: lang/crystal/Portfile [hunk 1]
```diff
@@ -2,10 +2,9 @@
 
 PortSystem          1.0
 PortGroup           github 1.0
-PortGroup           openssl 1.0
```
Comment (herbygillot) [inline lang/crystal/Portfile:5]: Does Crystal no longer use OpenSSL?

Comment (mohd-akram) [reply lang/crystal/Portfile:5]: It does, good catch, I forgot to add the `depends` line. No need for the PortGroup.

**Comment by herbygillot** [comment]
So the `openssl` portgroup is the preferred way to add OpenSSL support.  With it, you can control whether your project uses OpenSSL 1.1 or OpenSSL 3 (ie., `openssl.branch 1.1`).  Just linking straight to the `libssl.dylib` in the lib directory will give you OpenSSL 3.  I'm curious, is there a reason why you're removing the `openssl` portgroup?

**Comment by mohd-akram** [comment]
I added it previously because I thought I needed it to get the path to static OpenSSL libs, but there are symlinks for that in `/opt/local/lib`. This is how it was before, and it allows using libre SSL.

**Comment by herbygillot** [comment]
> I added it previously because I thought I needed it to get the path to static OpenSSL libs, but there are symlinks for that in `/opt/local/lib`. This is how it was before, and it allows using libre SSL.

Got it, good point about LibreSSL.

**Pull request merged**